src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/PrimarySite/index.js

```diff
@@ -71,6 +71,10 @@ const StepPrimarySetup = () => {
 				value: secondaryValue,
 			};
 			setCurrentOnboardingData( currentData );
+		} else if ( window?.nfdOnboarding?.currentFlow === 'ecommerce' ) {
+			currentData.data.siteType.primary.refers = 'slug';
+			currentData.data.siteType.primary.value = 'business';
+			setCurrentOnboardingData( currentData );
```

Comment:
I think we can add it in `includes/Services/FlowService.php` in `update_data_for_ecommerce`